Client/src/Components/EventCreation/FetchAltText.jsx

```diff
@@ -8,16 +8,16 @@ function FetchAltText({ eventId }) {
     useEffect(() => {
         const timer = setTimeout(() => {
             setShowButton(true);
-        }, 10000);
+        }, 12000);
 
         return () => clearTimeout(timer);
     }, []);
     
     //function to fetch altText from GPT 
-    // let number = eventId;
+    let number = eventId;
     const fetchAltText = async () => {
         try {
-            const response = await fetch(`/altText/${eventId}`);
+            const response = await fetch(`http://localhost:1113/altText/${number}`);
```

Comment:
change to proxy

Reply:
Was doing testing to fix a proxy issue, but the proxy is setup.